modules/apps/headless/headless-admin-user/headless-admin-user-impl/rest-openapi.yaml

```diff
@@ -5,6 +5,61 @@ components:
             description:
                 An account represents an external account, for example a customer business.
             properties:
+                accountContactInformation:
+                    description:
+                        The account's contact information.
+                    properties:
+                        emailAddresses:
+                            description:
+                                A list of the account's email addresses, with one optionally marked as primary.
+                            items:
+                                $ref: "#/components/schemas/EmailAddress"
+                            type: array
+                        facebook:
+                            description:
+                                The account's Facebook account.
+                            type: string
+                        id:
+                            description:
+                                The ID of the `contactInformation`.
+                            format: int64
+                            readOnly: true
+                            type: integer
+                        jabber:
+                            description:
+                                The account's Jabber handle.
+                            type: string
+                        postalAddresses:
+                            description:
+                                A list of account's postal addresses, with one optionally marked as primary.
+                            items:
+                                $ref: "#/components/schemas/PostalAddress"
+                            type: array
+                        skype:
+                            description:
+                                The account's Skype handle.
+                            type: string
+                        sms:
+                            description:
+                                The account's SMS number.
+                            type: string
+                        telephones:
```

Comment:
this shoule be phones

Reply:
It's the same used for `organizationContactInformation` or `userAccountContactInformation` in the same file